node/lib/DeviceAccessor.ts

```diff
@@ -1,18 +1,89 @@
 
 import { DeviceInterface } from "./DeviceInterface";
-import { IDevice } from "./ITranslator";
+import { ITranslator } from "./ITranslator";
 
 import { EventEmitter } from "events";
 
 /**
  * Provides reflection-style access to device properties and methods via device interfaces.
  */
 export class DeviceAccessor {
```

Comment:
Yes. I expect the onboarding design to add a few properties and methods to some of these classes. But the design is not clear enough yet (in my mind at least) to even define stubs.

---
In reply to: [73274899](https://github.com/openT2T/opent2t/pull/2#discussion_r73274899) [](ancestors = 73274899)